benchmarks/run.sh

```diff
@@ -2,22 +2,49 @@
 
 set -xe
 
-WORKSPACE_ROOT="$( cd "$(dirname "$0")/.." ; pwd -P )"
-cd "$WORKSPACE_ROOT"
+function build_benchmarks () {
+  function build_benchmark () {
+    if [ $1 == true ]; then OUTPUT_DIR="dist/size/$2/main"; else OUTPUT_DIR="dist/size/$2"; fi
```

Comment:
We control this input, but I would still recommend quoting.
Additionally in bash's test (`[`) string comparison is done with a single `=`.

```suggestion
    if [ "$1" = "true" ]; then OUTPUT_DIR="dist/size/$2/main"; else OUTPUT_DIR="dist/size/$2"; fi
```